senders/pagerduty/send.go

```diff
@@ -17,10 +17,17 @@ const summaryMaxChars = 1024
 // SendEvents implements Sender interface Send
 func (sender *Sender) SendEvents(events moira.NotificationEvents, contact moira.ContactData, trigger moira.TriggerData, plot []byte, throttled bool) error {
 	event := sender.buildEvent(events, contact, trigger, plot, throttled)
-	_, err := pagerduty.ManageEvent(event)
+	event.Action = "trigger"
+	if events.GetSubjectState() == moira.StateOK {
+		event.Action = "resolve"
+	}
+	event.DedupKey = trigger.ID
+
+	res, err := pagerduty.ManageEvent(event)
```

Comment:
What is the reason you added a `res` variable declaration here?

Reply:
Ah this is typo